tests/test_basic_ops_dict.py

```diff
@@ -613,7 +620,7 @@ def test_creation_with_invalid_key() -> None:
 def test_set_with_invalid_key() -> None:
     cfg = OmegaConf.create()
     with pytest.raises(KeyValidationError):
-        cfg[object()] = "a"  # type: ignore
+        cfg[object()]  # type: ignore
```

Comment:
The `test_set_with_invalid_key` test is now identical to the `test_get_with_invalid_key` that follows it.
I would suggest to revert this change. (The tests still pass after reverting)

```suggestion
        cfg[object()] = "a"  # type: ignore
```

Reply:
good catch.
I fixed the tests to call `__getitem__` and `__setitem__` explicitly.
I didn't initially realize that those two tests actually were different at a glance.